.scalafmt.conf

```diff
@@ -0,0 +1,9 @@
+version = "3.5.9"
+runner.dialect = scala212
+align.preset = none
+optIn = {
+  configStyleArguments = false
+}
+docstrings.style = Asterisk
+maxColumn = 120
+indent.withSiteRelativeToExtends = 2
```

Comment:
I'm a big fan of auto-formatting and these are settings for the `scalafmt` formatter. IDEA can be set to use scalafmt instead of its own formatting rules. We don't need to use it, but if we do we should have the same settings.